doc/source/tutorial/stats/continuous_laplace_asymmetric.rst

```diff
@@ -0,0 +1,38 @@
+
+.. _continuous-laplace:
+
+Asymmetric Laplace Distribution
+================================================================
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*}
+    F(x, \kappa) & = & 1-\frac{\kappa^{-1}}{\kappa+\kappa^{-1}}\exp(-x\kappa),\quad x\ge0; \\
+                 & = & \frac{\kappa}{\kappa+\kappa^{-1}}\exp(x/\kappa),\quad x<0. \\
+    f(x, \kappa) & = & \frac{1}{\kappa+\kappa^{-1}}\exp(-x\kappa),\quad x\ge0; \\
+                 & = & \frac{1}{\kappa+\kappa^{-1}}\exp(x/\kappa),\quad x<0.
+    \end{eqnarray*}
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*}
+    \mu & = & \kappa^{-1}-\kappa\\
+    \sigma^2 & = & \kappa^{-2}-\kappa^2\\
+    \tilde\mu_3 & = & \frac{2(1-\kappa^6)}{(1+\kappa^4)^{3/2}}
+    \end{eqnarray*}
+
+
+References
+----------
+
+-  "Asymmetric Laplace distribution", Wikipedia
+   https://en.wikipedia.org/wiki/Asymmetric_Laplace_distribution
+
+-  Kozubowski TJ and Podg\'orski K, "A Multivariate and Asymmetric
```

Comment:
This is not rendering correctly:
![image](https://user-images.githubusercontent.com/6570539/99756212-7bcbff00-2aa1-11eb-92df-301ac0d9f5d0.png)
Maybe you need to start the docstring with `r"""`?

Reply:
I'm not what you mean—this is an `.rst` file, not ` Python script. I replaced this with the UTF-8 ó and hope that it works.

Reply:
This file still shows `Podg\'orski`.  Did you push the change in a commit?

Reply:
Just pushed all my commits.